cmd/podman/common/create.go

```diff
@@ -348,7 +348,7 @@ func DefineCreateFlags(cmd *cobra.Command, cf *entities.ContainerCreateOptions,
 		_ = cmd.RegisterFlagCompletionFunc(podIDFileFlagName, completion.AutocompleteDefault)
 		createFlags.BoolVar(
 			&cf.Privileged,
-			"privileged", false,
+			"privileged", podmanConfig.ContainersConfDefaultsRO.Containers.Privileged,
```

Comment:
I would prefer to only do this on Local case and then do the check on the server side if the user did not set it.

Reply:
That's what we're doing for all other options.  We recently talked about the differences between podman and podman-system-service not having the same default settings.  But I don't think this PR is the right one to address it.  We need a general overhaul of how containers.conf is plumbed into Podman (and how the specgen works).

Reply:
Some are done correctly and others are not, adding another one done incorrectly, it not moving towards fixing the problem.

Reply:
I do not think any of those are done correctly on the REST API.

It remains largely unclear which settings should be set on the client and which should be set on the server - or both?  Until there is a decision I really don't want to just do something but remain consistent.

Reply:
I updated _without_ your requested changes but will take the next 30 minutes exploring a bit whether it's actually possible to distinguish on the REST API whether a given field has been explicitly set or not. We through the entire specgen over the fence and do not use http paramaters.

Reply:
OK, I got confirmation.  The REST API is not capable to distinguish an unset field from explicitly turned off field.

So if the REST handler would use the privileged settings from containers.conf and I'd run `podman-remote create --privileged=false [...]` the container would still be privileged.

As mentioned above, the code's not ready for what we want to do.